themes/theme-gmd/locale/de-DE.json

```diff
@@ -507,30 +507,30 @@
   },
   "login": {
     "headline": "Willkommen!",
-    "subline": "Bitte melde dich an, um fortzufahren",
+    "subline": "Bitte melden Sie sich an, um fortzufahren",
     "email": "E-Mail",
     "password": "Passwort",
     "forgot_password": "Passwort vergessen?",
-    "forgot_password_popup": "Bitte benutze die Desktop-Website, um Dein Passwort zurückzusetzen.",
+    "forgot_password_popup": "Bitte benutzen Sie die Desktop-Website, um Ihr Passwort zurückzusetzen.",
     "button": "Anmelden",
-    "no_account": "Du hast noch kein Konto?",
-    "register": "Starte hier.",
-    "logout_message": "Du wurdest ausgeloggt.",
-    "auto_logout_message": "Du wurdest automatisch ausgeloggt.",
+    "no_account": "Sie haben noch kein Konto?",
+    "register": "Starten Sie hier.",
+    "logout_message": "Sie wurden ausgeloggt.",
+    "auto_logout_message": "Sie wurden automatisch ausgeloggt.",
     "error": "Nutzername oder Passwort sind nicht korrekt",
     "reset_password": {
-      "subline": "Bitte gib deine Email-Adresse ein.",
-      "reset_instructions": "Du erhälst eine Email zum Zurücksetzen deines Passworts an {email}",
+      "subline": "Bitte geben Sie Ihre E-Mail-Adresse ein.",
+      "reset_instructions": "Sie erhalten eine E-Mail zum Zurücksetzen Ihres Passworts an {email}",
       "back_to_login": "Zurück zur Anmeldung",
-      "customer_not_found": "Zu dieser Email Adresse liegt und kein Account vor."
+      "customer_not_found": "Zu dieser E-Mail Adresse liegt und kein Account vor."
```

Comment:
correct spelling in German is E-Mail-Adresse with dashes because it's a Kompositum

Reply:
seems that you've already updated that